templates/components/products/card.html

```diff
@@ -1,3 +1,5 @@
+{{inject "isProductCardPresented" true}}
```

Comment:
Maybe we should add the same flag to `list-item component` since there is a possibility it can be used as a `cart component` (without list) in the future

Reply:
Since it used only in scope of list I prefer to leave as it is